### PR TITLE
AB#91770 FrontDoor setup

### PR DESCRIPTION
**Pull request by Carq**

### CODE REVIEW

**Review by Antek012 — APPROVED** [review]

File: src/HE.Investments.FrontDoor.Domain/HE.Investments.FrontDoor.Domain.csproj [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <RootNamespace>HE.Investments.Assessment.Domain</RootNamespace>
```
Comment (jjankowski87) [inline src/HE.Investments.FrontDoor.Domain/HE.Investments.FrontDoor.Domain.csproj:5]: remove this and fix namespace in files (where needed)

File: src/HE.Investments.FrontDoor.WWW/Routing/AssessmentErrorViewPaths.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+using HE.Investments.Common.WWW.Infrastructure.ErrorHandling;
+
+namespace HE.Investments.FrontDoor.WWW.Routing;
+
+public class AssessmentErrorViewPaths : IErrorViewPaths
```
Comment (jjankowski87) [inline src/HE.Investments.FrontDoor.WWW/Routing/AssessmentErrorViewPaths.cs:5]: Assessment -> FrontDoor

File: src/HE.Investments.FrontDoor.WWW/wwwroot/javascripts/land-cost-radio.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+function setUpBuyLandRadio() {
```
Comment (jjankowski87) [inline src/HE.Investments.FrontDoor.WWW/wwwroot/javascripts/land-cost-radio.js:1]: hmm, do we need all js files? this seems like specific for single view

**Review by jjankowski87 — APPROVED** [review]

File: src/HE.Investments.FrontDoor.WWW/wwwroot/javascripts/land-cost-radio.js (hunk 3 above)
Comment (Carq) [reply src/HE.Investments.FrontDoor.WWW/wwwroot/javascripts/land-cost-radio.js:1]: Removed.

**Pull request merged**